LaTeX/DocStrip.sublime-syntax

```diff
@@ -0,0 +1,97 @@
+%YAML 1.2
+---
+# https://www.sublimetext.com/docs/syntax.html
+name: TeX (DocStrip)
+file_extensions:
+  - ins
+scope: source.tex.docstrip
+
+extends: Packages/LaTeX/TeX.sublime-syntax
+
+contexts:
+  main:
+    - meta_prepend: true
+    - include: docstrip-preamble
+    - include: docstrip-config
+    - include: docstrip-user-io
+    - include: docstrip-keywords
+    - include: docstrip-constants
+
+  controls:
+    - meta_append: true
+    - match: ((\\)(?:endbatchfile)){{endcs}}
+      captures:
+        1: keyword.control.tex.docstrip
+        2: punctuation.definition.backslash.tex
```

Comment:
Maybe cleanup some unused capture groups such as ...

```suggestion
    - match: (\\)endbatchfile{{endcs}}
      scope: keyword.control.tex.docstrip
      captures:
        1: punctuation.definition.backslash.tex
```

Applies to various other patterns in this syntax as well.